### PR TITLE
[AIG] Onboarding updates

### PR DESCRIPTION
**Pull request by kodster28**
Goals:
- Flatten the "Get started" 4 pages into a single, streamlined guide (and offload info elsewhere as appropriate)
- Clearly call out the "Next steps" that turn beginners into power users
- Provide a space where we can put in a more prescriptive tutorial(s) around onboarding with Workers AI

### CODE REVIEW

File: content/ai-gateway/_partials/_delete-gateway.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+---
+
+{{<tabs labels="Dashboard | API">}}
+{{<tab label="dashboard" no-code="true">}}
+
+To delete an AI Gateway in the dashboard:
+
+1. Log into the [Cloudflare dashboard](https://dash.cloudflare.com/) and select your account.
+2. Go to **AI** > **AI Gateway**.
+3. Select your gateway
```
Comment (marciocloudflare) [inline content/ai-gateway/_partials/_delete-gateway.md]:
```suggestion
3. Select your gateway.
```

File: content/ai-gateway/observability/analytics.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+{{<tab label="graphql" no-code="true">}}
+
+You can use GraphQL to query your usage data outside of the AI Gateway dashboard. See the example query below. You will need to use your Cloudflare token when making the request, and change `{account_id}` to match your account tag.
+
+```bash
+---
+header: Request
+---
+curl --request POST \
+  --url https://api.cloudflare.com/client/v4/graphql \
```
Comment (marciocloudflare) [inline content/ai-gateway/observability/analytics.md]:
```suggestion
curl https://api.cloudflare.com/client/v4/graphql \
```

Comment (marciocloudflare) [reply content/ai-gateway/observability/analytics.md]: POST requests with a body do not require `--request POST \` argument, per our guidelines

File: content/reference-architecture/diagrams/ai/ai-multivendor-observability-control.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ The following architecture illustrates the setup of [AI Gateway](/ai-gateway/) a
 ![Figure 1: Multi-vendor AI architecture](/images/reference-architecture/ai-multivendor-observability-control/ai-multi-vendor-observability-control.svg "Multi-vendor AI architecture")
 
 1. **Inference request**: Send POST request to your AI gateway.
-2. **Request proxying**: Forward POST request to AI Inference provider or serve response from [cache, if enabled and available](/ai-gateway/get-started/configuring-settings/#caching). Both [analytics](/ai-gateway/get-started/configuring-settings/#analytics) and [logs](/ai-gateway/get-started/configuring-settings/#logging) are collected in the process and controls such as Rate Limiting enforced.
+2. **Request proxying**: Forward POST request to AI Inference provider or serve response from [cache, if enabled and available](/ai-gateway/configuration/caching). Both [analytics](/ai-gateway/observability/analytics/) and [logs](/ai-gateway/observability/analytics/#logging) are collected in the process and controls such as Rate Limiting enforced.
```
Comment (marciocloudflare) [inline content/reference-architecture/diagrams/ai/ai-multivendor-observability-control.md]:
```suggestion
2. **Request proxying**: Forward `POST` request to AI Inference provider or serve response from [cache, if enabled and available](/ai-gateway/configuration/caching). Both [analytics](/ai-gateway/observability/analytics/) and [logs](/ai-gateway/observability/analytics/#logging) are collected in the process and controls such as Rate Limiting enforced.
```

**Review by marciocloudflare — APPROVED** [review]

**Comment by kodster28** [comment]
hyperlint review

**Pull request merged**